creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -913,15 +920,28 @@ class SnapbackSM {
       'BEGIN processStateMachineOperation()',
       {
         currentModuloSlice: this.currentModuloSlice,
-        moduloBase: this.moduloBase
+        moduloBase: this.moduloBase,
+        lastProcessedUserId: this.lastProcessedUserId
       }
     )
 
+    let nodeUsers = []
+    // New DN versions support pagination, so we fall back to modulo slicing for old versions
+    // TODO: Remove modulo supports once all DNs update to include https://github.com/AudiusProject/audius-protocol/pull/3071
```

Comment:
can we add this as a linear followup task? and what version of disc prov will allow us to remove this modulo support?

Reply:
assuming he didnt specify version since DP PR isn't yet merged, but +1 on linear followup task (due date feature is great to send you reminder notif later)